torch/csrc/jit/frontend/schema_matching.cpp

```diff
@@ -143,6 +143,14 @@ Value* tryConvertToType(
       } else if (concrete_int) {
         value = graph.insert(aten::Int, {value}, {}, loc);
       }
+    } else if (*value->type() == *BoolType::get()) {
```

Comment:
Can you also describe what this is doing? I'm not too familiar with this part of the codebase

Reply:
if some op just supports float/int, but the user input is bool, then add an implicit conversion from bool to float/int.
this is the actual fix for issues mentioned in the pr description.

Reply:
I'll try to find a good place to add the extra test for jit script.